public/app/features/canvas/elements/server.tsx

```diff
@@ -0,0 +1,474 @@
+            <g>
+              <path
+                className={styles.server}
+                x="54.86203"
+                y="48.088943"
+                width="159.1676"
+                height="39.961231"
+                d="m56.979 48.089h54.93c1.169 0 2.1167.94768 2.1167 2.1167v145.73c0 1.169-.94768 2.1167-2.1167 2.1167h-54.93c-1.169 0-2.1167-.94768-2.1167-2.1167v-145.73c0-1.169.94768-2.1167 2.1167-2.1167z"
+              />
+              <g transform="matrix(.50833 0 0 1 27.315 0)">
+                <rect x="55.556" y="57.472" width="107.76" height="5.7472" />
+                <rect x="55.558" y="86.141" width="107.76" height="5.7472" />
+                <g className={styles.thinLine}>
+                  <rect x="74.357" y="73.261" width="96.957" height="2.8736" />
+                  <rect x="74.379" y="101.9" width="96.957" height="2.8736" />
+                </g>
+              </g>
+              <circle
+                className={styles.circle}
+                cx="83.858"
+                cy="178.2"
+                r="5.9401"
+                style={{ animation: bulbAnimation, fill: bulbColor }}
+              />
+            </g>
+            <g transform="translate(-1.9978 -7.5028)">
+              <path className={styles.monitor} d="m103.22 75.305h118.87v76.914h-118.87z" />
+              <path
+                className={styles.monitorOutline}
+                d="m103.22 70.305-5 5v76.916l5 5h118.87l5-5v-76.916l-5-5zm5 10h108.87v58.916h-108.87z"
+              />
+            </g>
+            <path d="m135.6 148.14h50.113l4 15.156h-58.113z" />
+            <path className={styles.keyboard} d="m118.23 183.19h88.848l24 19.476h-136.85z" />
+          </g>
+        </>
+      ) : null}
+    </svg>
+  );
+};
+
+export const serverItem: CanvasElementItem<ServerConfig, ServerData> = {
+  id: 'server',
+  name: 'Server',
+  description: 'Basic server with status',
+
+  display: ServerDisplay,
+
+  defaultSize: {
+    width: 100,
+    height: 155,
+  },
+
+  getNewOptions: (options) => ({
+    ...options,
+    background: {
+      color: {
+        fixed: 'transparent',
+      },
+    },
+    placement: {
+      width: options?.placement?.width ?? 100,
+      height: options?.placement?.height ?? 155,
+      top: options?.placement?.top,
+      left: options?.placement?.left,
+    },
+  }),
+
+  // Called when data changes
+  prepareData: (ctx: DimensionContext, cfg: ServerConfig) => {
+    const data: ServerData = {
+      blinkRate: cfg?.blinkRate ? ctx.getScalar(cfg.blinkRate).value() : 0,
+      statusColor: cfg?.statusColor ? ctx.getColor(cfg.statusColor).value() : 'transparent',
+      bulbColor: cfg?.bulbColor ? ctx.getColor(cfg.bulbColor).value() : 'green',
+      type: cfg?.type ?? ServerTypes.Single,
+    };
+
+    return data;
+  },
+
+  registerOptionsUI: (builder) => {
+    const category = ['Server'];
+    builder
+      .addRadio({
```

Comment:
Maybe a drop down for this might be a little more responsive / allow for easier extensibility in the future?
![Screenshot 2023-01-11 at 1 43 32 PM](https://user-images.githubusercontent.com/22381771/211924228-8a80ef6c-2da7-4d34-98a0-8f50286a6b69.png)